sql/migrations/20220601082200_logs.sql

```diff
@@ -0,0 +1,48 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20220601082200');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20220601082200');
+-- Add your query below.
+
+-- Now unused tables
+DROP TABLE IF EXISTS `logs_behavior`;
+DROP TABLE IF EXISTS `logs_characters`;
+DROP TABLE IF EXISTS `logs_chat`;
+DROP TABLE IF EXISTS `logs_movement`;
+DROP TABLE IF EXISTS `logs_spamdetect`;
+DROP TABLE IF EXISTS `logs_warden`;
+
+-- New unified table
+DROP TABLE IF EXISTS `logs_player`;
+CREATE TABLE `logs_player` (
+  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `time` DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
**SQL Error (1067): Invalid default value for 'time'**

The plan in vmangos is to never raise the minimum requirements, and this part does not work in mysql 5.5, which is the last mysql version to run on Windows XP. 

Suggested fix:
```
`time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Reply:
Resolved by [e8687a6](https://github.com/vmangos/core/pull/1519/commits/e8687a62d119f6a32cc02c4b7c5f4f277336ab2f).